ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5193,3 +5193,75 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+- name: programBulkJobRequestCreate
+    request_path: "{{ program_service_prefix }}/v1/process/create"
+    upstream_url: "{{ program_service_url }}/program/v1/process/create"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'appAccess'
```

Comment:
invalid Access control list @venkateshwarans please check all the below apis as well, they have same issues.